azurerm/internal/services/netapp/netapp_volume_resource_test.go

```diff
@@ -248,6 +264,62 @@ resource "azurerm_netapp_volume" "test" {
 `, template, data.RandomInteger, data.RandomInteger)
 }
 
+func (NetAppVolumeResource) crr(data acceptance.TestData) string {
```

Comment:
could we expand crr to be unabbreviated

Reply:
Sure, replaced crr with crossRegionReplication